data/wikipron/src/scrape.py

```diff
@@ -45,19 +79,65 @@ def _call_scrape(
         lang_settings["key"],
         lang_settings,
     )
+    # Checks if second TSV was opened.
+    if os.path.exists(tsv_path_filtered):
+        os.remove(tsv_path_filtered)
     os.remove(tsv_path)
 
 
 def _build_scraping_config(
     config_settings: Dict[str, Any], wiki_name: str, dialect_suffix: str = ""
 ) -> None:
     path_affix = f'../tsv/{config_settings["key"]}_{dialect_suffix}'
+    whitelist_path_affix = (
+        f"../whitelist/{config_settings['key']}_{dialect_suffix}"
+    )
+
+    # Configures phonemic TSV
```

Comment:
Period at end of comment (and below) please. (I don't have a style guide to point to this but it's how we've done things.)